letsencrypt/data/run.sh

```diff
@@ -15,6 +21,7 @@ mkdir -p "$CERT_DIR"
 mkdir -p "/ssl"
 chmod +x /run.sh
 touch /data/dnsapikey
+PROVIDER_ARGUMENTS=""
```

Comment:
`PROVIDER_ARGUMENTS=()`

Reply:
Updated as requested

Reply:
Hey @pvizeli, random bystander here.  What do the parenthesis do in this case?  I was unfamiliar with this usage of bash.  It looks like it executes the contents in a subshell.